src/docs/sdk/sessions.mdx

```diff
@@ -256,13 +258,29 @@ application run. This is what you would see in a web browser, mobile world,
 command line application or similar.
 
 - typically just a single session from application start to quit
-- there are usually fewer sessions than events
+- there are usually fewer sessions than captured events
 - sessions span multiple hubs / threads
 - session duration typically in the minutes, timing information is useful
 
 Both of those cases look similar from the API point of view but different
 recommendations apply for SDKs.
 
+### Choosing the Session Mode
+
+While it is in theory possible to use both session modes in a single application,
+it is recommended that the SDK defaults to a single mode that is most
+appropriate to the main use-case of the language ecosystem. This is similar to,
+and can be used in the same way as a *global Hub mode* that certain SDKs support.
+
+When the SDK is configured to use *user-mode* sessions, or global Hub mode, a
+single session should be started at the start of the application and should
+persist through the applications runtime. Depending on the SDK internals, this
+single session can be shared among all application threads and thread-local Hubs.
+
+When using *server-mode* sessions, no application-wide session will be started,
+and it is up to integrations or the user to start and end the session for the
```

Comment:
```suggestion
and it is up to integrations or the user to start the session when the request is received and end it when a response is returned.
```

(Doing this all in one line (and getting rid of the line below) because it's too complicated otherwise. Really wish we didn't do manual line breaks in our docs...)